src/foam/nanos/crunch/predicate/IsAgent.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.crunch.predicate',
+  name: 'IsAgent',
```

Comment:
Is this used anywhere?

Reply:
Nanopay side - used in a predicate for signing officer privileges so it doesn't show up when your real user and effective user are the same